docs/src/1_Mechanics/04_Powers.yaml

```
@@ -947,37 +586,51 @@ Hidden Strike:
     Tags:
         - Assassin  
 
+Stealth in the Shadows:
+    Category:
+        - Movement
+        - Stealth
+    Description: A creature of the dark, you're especially adept at going unseen.
+    Mechanic: When you make a Stealth check in dim light or darkness, you draw with the Upper Hand.
+    Type: Passive
+    XP: 1
+    Prereq:
+        Skill: Stealth > 0
+        Level: 2
+    Tags: 
+    - Stealth
+    - Assassin
+
 Vanishing Step: 
     Type: Major
     Category: 
         - Movement
         - Stealth
-    Mechanic: Choose a point you can see within 10 spaces of you. If the space is unoccupied, you appear there Hidden. You cannot attempt to occupy a space that is occupied by another creature.
+    Mechanic: Choose a point you can see. If the space is unoccupied, you appear there Hidden. You cannot attempt to occupy a space that is occupied by another creature.
```

In its current state, range would not appear as part of this on a char sheet. For now, I would add it back here

Is 'Hidden' a status effect? Is it purposefully capitalized?
```suggestion
    Mechanic: Choose a point you can see within 10 spaces. If the space is unoccupied, you appear there Hidden. You cannot attempt to occupy a space that is occupied by another creature.
```

---

We're using `Hidden` in more places across the Assassin skill-set, with Hidden Strike and Vanishing Step both using it, so maybe it's worth us looking at where it should go if we want to keep it as a capitalized mechanic. I think it operates mechanically similar to `Cover` and `Lighting` as opposed to being a Status Effect as it only affects the creatures attempting to see the Hidden creature rather than it being something that affects the Dealer themselves.